README.md

```diff
@@ -424,7 +424,7 @@ Fill in a combo box and select an option
   - `from` [String, Symbol, Array] - Locator for the field
   - `search` [String] - Alternative text to search for in the input
   - `currently_with` [String] - Current value for the field
-  - options prefixed with 'option_' will be used to find the option. eg `option_text`, `option_match`
+  - options prefixed with 'option\_' will be used to find the option. eg `option_text`, `option_match`
```

Comment:
probably worth just using double quotes here if you want to be stringent.

Reply:
Cool thanks. I will set my editor up so it tries to retain the original markdown formatting and push a change. I've only just started to explore it but looks like a great library and interested to see it grow. Have you considered publishing it to RubyGems?

Reply:
@luke-hill I've fixed that other typo you flagged to insert backticks for code highlighting. VSCode makes it frustratingly hard to prevent it automatically turning 2 spaces after a full stop to 1. So unfortunately the other formatting changes introduced my editor may have to stick.

I kept it to a single commit but updated the message so if you integrate this you don't pollute the git history with unnecessary commits.

Reply:
RE deploying to rubygems. We currently have a couple of public repos which we're happy to do so on. We just need to agree "how" we do this. It's something we're definitely wanting to do, and we do so I believe for our npm packages (For those that are public). I think the versioning / branching is good enough for now, but yes I agree it's not an optimal solution.